ion/symboltable.go

```diff
@@ -128,6 +129,20 @@ func (s *sst) Adjust(maxID uint64) SharedSymbolTable {
 	}
 }
 
+func (s *sst) Find(sym string) *SymbolToken {
+	id, ok := s.FindByName(sym)
+	if !ok {
+		return nil
+	}
+
+	text, ok := s.FindByID(id)
+	if !ok {
+		return nil
+	}
+
+	return &SymbolToken{Text: &text, LocalSID: SymbolIDUnknown}
```

Comment:
Why are we setting the SID to `SymbolIDUnknown`? What about the `id` that we computed earlier?

Reply:
According to the Ion specs, it is irrelevant

> SymbolToken APIs, return a SymbolToken with that text and with an undefined importLocation (it is irrelevant).

http://amzn.github.io/ion-docs/guides/symbols-guide.html

Reply:
According to Almann: https://github.com/amzn/ion-go/issues/11#issuecomment-580951115

> If the Local SID is defined, the text must be unknown and the source must be undefined. This implies that the symbol token has yet to be resolved and we do not (yet) know the context of such a token.
In all other cases Local SID must be undefined as this implies that the symbol token has been resolved and the local identifier is not meaningful in this context.